agent/src/dbus/softwarecontaineragent_dbus_stub.cpp

```diff
@@ -227,51 +242,51 @@ void com::pelagicore::SoftwareContainerAgent::on_method_call(const Glib::RefPtr<
     }
 }
 
-void com::pelagicore::SoftwareContainerAgent::on_interface_get_property(Glib::VariantBase& property,
-                      const Glib::RefPtr<Gio::DBus::Connection>& connection,
-                      const Glib::ustring& sender,
-                      const Glib::ustring& object_path,
-                      const Glib::ustring& interface_name,
-                      const Glib::ustring& property_name) {
-
+void com::pelagicore::SoftwareContainerAgent::on_interface_get_property(
+    Glib::VariantBase& /* property */,
+    const Glib::RefPtr<Gio::DBus::Connection>& /* connection */,
+    const Glib::ustring& /* sender */,
+    const Glib::ustring& /* object_path */,
+    const Glib::ustring& /* interface_name */,
+    const Glib::ustring& /* property_name */)
+{
 }
 
 bool com::pelagicore::SoftwareContainerAgent::on_interface_set_property(
-       const Glib::RefPtr<Gio::DBus::Connection>& connection,
-       const Glib::ustring& sender,
-       const Glib::ustring& object_path,
-       const Glib::ustring& interface_name,
-       const Glib::ustring& property_name,
-       const Glib::VariantBase& value) {
-
-
+    const Glib::RefPtr<Gio::DBus::Connection>& /* connection */,
+    const Glib::ustring& /* sender */,
+    const Glib::ustring& /* object_path */,
+    const Glib::ustring& /* interface_name */,
+    const Glib::ustring& /* property_name */,
+    const Glib::VariantBase& /* value */)
+{
     return true;
 }
 
-void com::pelagicore::SoftwareContainerAgent::ProcessStateChanged_emitter(gint32 containerID, guint32 processID, bool isRunning, guint32 exitCode) {
-            std::vector<Glib::VariantBase> paramsList;
-
-paramsList.push_back(Glib::Variant<gint32 >::create((containerID)));;
-
-
-paramsList.push_back(Glib::Variant<guint32 >::create((processID)));;
-
-
-paramsList.push_back(Glib::Variant<bool >::create((isRunning)));;
-
-
-paramsList.push_back(Glib::Variant<guint32 >::create((exitCode)));;
+void com::pelagicore::SoftwareContainerAgent::ProcessStateChanged_emitter(
+    gint32 containerID,
+    guint32 processID,
+    bool isRunning,
+    guint32 exitCode)
+{
+    std::vector<Glib::VariantBase> paramsList;
+    paramsList.push_back(Glib::Variant<gint32 >::create((containerID)));;
+    paramsList.push_back(Glib::Variant<guint32 >::create((processID)));;
+    paramsList.push_back(Glib::Variant<bool >::create((isRunning)));;
+    paramsList.push_back(Glib::Variant<guint32 >::create((exitCode)));;
 
-m_connection->emit_signal(
-              "/com/pelagicore/SoftwareContainerAgent",
-              "com.pelagicore.SoftwareContainerAgent",
-              "ProcessStateChanged",
-              Glib::ustring(),
-              Glib::Variant<std::vector<Glib::VariantBase> >::create_tuple(paramsList));
-      }
+    m_connection->emit_signal(
```

Comment:
I think my editor converts tabs to spaces for me, plus I think everything was spaces already.